src/index.tsx

```diff
@@ -24,6 +26,8 @@ const container = document.getElementById(
   appServices.appConfiguration.appElementId
 );
 
+ReactModal.setAppElement("#modal-wrapper");
```

Comment:
The _app container element_ is configurable. Should we also make the _modal container element_ configurable?

(Imagine scenarios where more than one application is sharing the same HTML file)

Reply:
Yes is configurable but react-modal use the body to inyecte the `<modal/>`

![image](https://user-images.githubusercontent.com/6733401/214613424-eb656367-e397-421a-81df-3c841b115a56.png)

Reply:
Talking with @andresmoschini I decide use the prop ariaHideApp of react-modal, with this property we avoid use the SetAppElement method. This is a better aproach because the html's safe in scenaries where the html is shared between apps.

Reply:
I do not understand how it would help when the HTML is shared between apps. Let's talk about it.

Reply:
We talk about this and confirm that using _Aria_ and `SetAppElement` is a good thing and @Cromeror will try to apply it.

Reply:
Summary, we use a ID in the app element wrapper to shared this container between different apps